test/UniswapV2Router01.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('UniswapV2Router01', () => {
     WETH = fixture.WETH
     WETHPartner = fixture.WETHPartner
     factory = fixture.factoryV2
-    router = fixture.router
+    router = fixture.router01
```

Comment:
you could avoid copying the entire test suite if you just run this test suite for both routers, e.g.

```typescript
const routers = [fixture.router01, fixture.router02]
// ...
for (const ix in [router01, router02]) {
  describe(`router v${ix+1}`, () => {
  const router = 
    // ...
  })
}
```

that way we know what's unique to the v2 router

Reply:
done